src/index.js

```diff
@@ -144,7 +151,13 @@ export class WT {
       delete this.loaderImage.onload;
       reject();
     };
-    this.loaderImage.src = `${this.getUrl()}?${query}`;
+    const params = new this.context.URLSearchParams(query);
```

Comment:
along the lines of brad's browser support callout, this is a newer API. we already use the `qs` library so i'd suggest using that (or fully migrating to `URLSearchParams` if you have time for a bigger change).

Reply:
What's our cutoff for browser support for both `sendBeacon` and `URLSearchParams`?  Their browser compatibility is pretty similar with full support on latest versions except no support on IE. For `sendBeacon`'s data parameter, I'm taking advantage of the `URLSearchParams` object so they kind of go hand-in-hand so I don't think I could use `qs` with `sendBeacon`. I also tried the [Blob](https://developer.mozilla.org/en-US/docs/Web/API/Blob) object with `sendBeacon` to send the data as JSON but that led to CORS issues. 

i selfishly like the idea that noah suggested to leave as is, add note in the readme that `sendBeacon` and `URLSearchParams` are required and let users polyfill if necessary. 

this also leads back to the fallback question: if we're requiring sendBeacon support, should we just get rid of all fallbacks (POST and image)? i left the image code in there to keep the change small